src/python/grpcio/grpc/_cython/loader.c

```diff
@@ -62,6 +63,18 @@ int pygrpc_load_core(char *path) { return 1; }
 
 #endif  /* !GPR_WINDOWS */
 
+//Cython doesn't have Py_AtExit bindings, so we call the C_API directly
```

Comment:
[Missing single space between `//` and comment prose](https://google.github.io/styleguide/cppguide.html#Implementation_Comments).